src/Raven.Studio/typescript/models/database/tasks/periodicBackup/amazonSettings.ts

```diff
@@ -84,13 +84,18 @@ abstract class amazonSettings extends backupSettings {
 
     initAmazonValidation() {
         const self = this;
+        
         this.awsRegionName.extend({
             required: {
-                onlyIf: () => this.enabled()
+                onlyIf: () => this.isRegionRequired()
             },
             validation: [
                 {
                     validator: function (awsRegionName: string) {
+                        if (!self.isRegionRequired()) {
```

Comment:
here is another self?

Reply:
In this case, I _haven't_ introduced a new `self`
It is used to access the right fields (see fields below also)
`this` would be the validator object

Reply:
I know it wasn't you but it would be nice to fix it. so please use arrow function instead. Then this will be bound correctly

Reply:
refactored